### DISCUSSION TITLE
Dump of N64 Saves not working

### DISCUSSION THREAD

**Discussion by djthoz91** [sanni/cartreader#876] [Need help]
Hello,

I tried to extract the ROMs and backups from my N64 cartridges. I was able to extract the ROMs from the cards. However, when I try to retrieve the EEPROMs from the cards, I only get eep files with an empty file. 

    The clock generator is installed

    I have put HW5 in the configuration

    I've run the self-test several times and the result has always been OK.

    I'm using version 13.1 

Thank you in advance for your help

Best regards,

Nico

**Comment by Hynes84** [comment]
Having the same issue on an Aliexpress unit. Hopefully someone can help

**Reply by Hynes84** [reply]
Update: Reverted to firmware V5.5 and was able to write eeprom successfully the "old" way (3.3V, EEP only)

**Reply by djthoz91** [reply]
do you have a hardware version 3? version 5.5 is an old version July 2020.
